db/migrations/00009_add_firmwares.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE component_firmware_version (
+  id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  component STRING NULL,
+  vendor STRING NULL,
+  model STRING NULL,
+  filename STRING NULL,
+  version STRING NULL,
+  utility STRING NULL,
+  sha STRING NULL,
```

Comment:
let's replace the word `sha` with `checksum` and not tie it to a specific algorithm. On the other hand... can the checksum actually be `null`? Aren't we always downloading some artifact and integrity checking it?

Reply:
Changed `sha` to `checksum` and changed all columns to be `NOT NULL` as part of Nicole's feedback above.